app/controllers/published_pages_controller.rb

```diff
@@ -31,6 +31,9 @@ def show
 
     @topic = pp.topic
     @canonical_url = @topic.url
+    @logo = SiteSetting.logo_small
```

Comment:
You might want to reuse the logic we have in application_helper:

```
  def application_logo_url
    @application_logo_url ||= begin
      if mobile_view? && SiteSetting.site_mobile_logo_url.present?
        SiteSetting.site_mobile_logo_url
      else
        SiteSetting.site_logo_url
      end
    end
  end
```

Reply:
I did try this out, but it returns the full width site logo on desktop. I want the small/mobile logo to appear here regardless of desktop/mobile user.

<kbd>
<img width="639" alt="image" src="https://user-images.githubusercontent.com/30537603/88930020-8a677f80-d240-11ea-86e3-042843eec5ca.png"></kbd>